outline/shadowsocks/config.go

```diff
@@ -15,18 +15,25 @@
 package shadowsocks
 
 import (
+	"encoding/json"
+	"errors"
+	"fmt"
+
 	"github.com/Jigsaw-Code/outline-ss-server/client"
 	"github.com/eycorsican/go-tun2socks/common/log"
 )
 
 // Config represents a shadowsocks server configuration.
-// Exported via gobind.
+// Exported via gobind.  Must match ShadowsocksSessionConfig interface in Typescript.
```

Comment:
Do we still need to export it via `gobind`? I think in TypeScript we don't need to access this data structure any more if we are going to use json strings.

Reply:
We can remove this, but it is currently part of outline-go-tun2socks's "public API", which we have decided not to break in this PR.

(The bindings exported by `gobind` do not reach Typescript.  They are used only by Java and Swift/ObjC.)